Units/parser-markdown.r/hashtags-utf8.d/input.md

```diff
@@ -0,0 +1,32 @@
+#标签 #태그 #وسم   
+#t1 #t2#invalid#invalid2
+
+123 is invalid
+#123
+
+#9_
+#-_-
+#t/s1 #t/s2
+#t4-5-6+excluded  #t7;excluded
+text#text #hashtag text#text
+#included
+    #excluded1
+	#excluded2
+   #included
```

Comment:
This comment is nothing to do with this pull request.
`#included` at line 15 is tagged as `c`hapter. This may be a bug of the original code.

Reply:
I opened #3748.